.circleci/config.yml

```diff
@@ -44,7 +44,8 @@ jobs:
     steps:
       - setup-dependencies
       - run: npm run test:unit
-      - run: npm run build
+      - run: npm run seed:development
```

Comment:
The seed:*** commands should run in the deployment steps, not in the build steps.

Reply:
done